client/src/routes/automation/BlocklyComponent.svelte

```diff
@@ -50,26 +50,30 @@
         );
 
         const rootBlock: string =
-            '<xml><block type="automation" deletable="false" movable="false"></block></xml>';
+            '<xml><block type="esidom_automation" deletable="false" movable="false"></block></xml>';
         Blockly.Xml.domToWorkspace(Blockly.Xml.textToDom(rootBlock), workspace);
 
         blocklyService = new BlocklyService(toolbox, workspace);
 
-        entityPromise = BlocklyService.createEntities();
+        entityPromise = BlocklyService.initBlockly();
     });
 </script>
 
 <div>
     {#await entityPromise}
-        <p>Chargement de blockly ...</p>
+        <p>Chargement de Blockly ...</p>
```

Comment:
Remember that you can now use the i18n lib to put those french strings in an external file

Reply:
Done.
All strings will be changed with time.